### PR TITLE
Update ProjectWrapper.tsx

### PR DESCRIPTION
**Pull request by ryanyae**
## Description

[Please enter a brief description of the PR]

Updated members in ProjectWrapper for IDEO

[Please enter a brief summary of updates to this PR during code review - if any]

## Checklist

- [ ] Ran `npm run lint:fix` to format code
- [ ] Requested at least one reviewer
- [ ] Connected PR to Trello ticket (steps below)
- [ ] Addressed code review comments
- [ ] Gained at least one approval for your PR

## Connecting your PR to the corresponding ticket:
- Click on the "GitHub" button found on the right of your Trello ticket
- Choose "Attach PR" from the dropdown list
- Link your GitHub account (if you haven't done so already)
- Navigate to`VCL-content-platform` and choose your PR from the dropdown.

### CODE REVIEW

File: src/pages/Project/ProjectWrapper.tsx [hunk 1]
```diff
@@ -552,14 +547,19 @@ const ProjectWrapper: React.FC<ProjectProps> = ({ match }) => {
...
           isCurrentMember: true,
         },
         {
           name: "Yebin Cho",
           position: "Co-pilot",
           isCurrentMember: true,
         },
+        {
+          name: 'Amira Dakkak',
+          position: 'Position',
```
Comment (ElisaHu1) [inline src/pages/Project/ProjectWrapper.tsx:560]: do we not know this person's position?

File: src/pages/Project/ProjectWrapper.tsx [hunk 2]
```diff
@@ -552,14 +547,19 @@ const ProjectWrapper: React.FC<ProjectProps> = ({ match }) => {
         },
         {
           name: "Astrid Tam",
-          position: "Co-pilot",
+          position: "Volunteer",
           isCurrentMember: true,
         },
         {
           name: "Yebin Cho",
```
Comment (ElisaHu1) [inline src/pages/Project/ProjectWrapper.tsx:554]: also to maintain the order, can we move the co-pilot before volunteer?

File: src/pages/Project/ProjectWrapper.tsx (hunk 1 above)
Comment (ryanyae) [reply src/pages/Project/ProjectWrapper.tsx:560]: Oh we should, I must have forgot.

**Comment by ryanyae** [comment]
[IDEO people update](https://trello.com/c/9alW0P7v/189-ideo-people-update)

**Review by ElisaHu1 — APPROVED** [review]

**Pull request merged**